pom.xml

```diff
@@ -70,6 +70,10 @@
         <!-- Replaced by the build number plugin at build time: -->
         <buildNumber>${user.name}-${maven.build.timestamp}</buildNumber>
         <jacoco.skip>true</jacoco.skip>
+        <!--suppress CheckTagEmptyBody -->
+        <surefire.argLine></surefire.argLine>
```

Comment:
I'm guessing this works, but (if you haven't already) double-check to make sure the code coverage works (Jacoco sets the `argLine` prop too)